nodejs/week1/homework/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "minimist": "^1.2.8"
```

Comment:
If this dependency is not used anymore, it should be removed.

Reply:
done